catalog/ui/catalog-ui-search-api/pom.xml

```diff
@@ -0,0 +1,119 @@
+                <configuration>
+                    <!-- Bulk XML test resources more concise without the header -->
+                    <excludes>src/test/resources/forms/</excludes>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Import-Package>
+                            ddf.catalog.data,
+                            *
+                        </Import-Package>
+                        <Export-Package>
+                            org.codice.ddf.catalog.ui.metacard.workspace,
+                            org.codice.ddf.catalog.ui.metacard.workspace.transformer
+                        </Export-Package>
+                        <Embed-Dependency>
+                            catalog-core-api-impl,
+                            platform-util
+                        </Embed-Dependency>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-check</id>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <configuration>
+                            <haltOnFailure>true</haltOnFailure>
+                            <rules>
+                                <rule>
+                                    <element>BUNDLE</element>
+                                    <limits>
+                                        <limit>
+                                            <counter>INSTRUCTION</counter>
+                                            <value>COVEREDRATIO</value>
+                                            <minimum>0.33</minimum>
+                                        </limit>
+                                        <limit>
+                                            <counter>BRANCH</counter>
+                                            <value>COVEREDRATIO</value>
+                                            <minimum>0.28</minimum>
+                                        </limit>
+                                        <limit>
+                                            <counter>COMPLEXITY</counter>
+                                            <value>COVEREDRATIO</value>
+                                            <minimum>0.29</minimum>
+                                        </limit>
+                                        <limit>
+                                            <counter>LINE</counter>
```

Comment:
I don't think we use `line` any more; it should be removed from `catalog-ui-search` but that will be out of scope for this PR.

Reply:
Removed!